<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>573</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>573</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 111, 166);
border-top-left-radius:5px;
border-top-right-radius:5px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>THÊM NHÂN VIÊN</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-bottom-left-radius:5px;
border-bottom-right-radius:5px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_6">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Họ và tên</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="name_input">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>-1</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#name_input{
	border: 2px solid rgb(226, 226, 226);
	border-radius:3px;
	font: 16px &quot;Segoe UI&quot;;
}
#name_input:focus{
	border: 2px solid rgb(0, 170, 255);        
    background: #ffffff;  
}</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_8">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Email</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="email_input">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>-1</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#email_input{
	border: 2px solid rgb(226, 226, 226);
	border-radius:3px;
	font: 16px &quot;Segoe UI&quot;;
}
#email_input:focus{
	border: 2px solid rgb(0, 170, 255);        
    background: #ffffff;  
}</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_10">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Địa chỉ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="address_input">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>-1</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">#address_input{
	border: 2px solid rgb(226, 226, 226);
	border-radius:3px;
	font: 16px &quot;Segoe UI&quot;;
}
#address_input:focus{
	border: 2px solid rgb(0, 170, 255);        
    background: #ffffff;  
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_11">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_13">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Giới tính</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="sex_comboBox">
                 <property name="minimumSize">
                  <size>
                   <width>130</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>-1</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QComboBox#sex_comboBox {
    border: 2px solid rgb(226, 226, 226);    
    border-radius: 3px;           
    padding: 6px 10px;             
    background: #fdfdfd;           
    font: 16px &quot;Segoe UI&quot;;
}

QComboBox#sex_comboBox:focus {
    border: 2px solid rgb(0, 170, 255);        
    background: #ffffff;           
}

QComboBox#sex_comboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border-left: 1px solid #ccc;
}

QComboBox#sex_comboBox::down-arrow {
	image: url(:/UI/icons/chevron-down.svg);
    width: 12px;
    height: 12px;
}
</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>Nam</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Nữ</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_14">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Số điện thoại</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="phonenumber_input">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>-1</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#phonenumber_input{
	border: 2px solid rgb(226, 226, 226);
	border-radius:3px;
	font: 16px &quot;Segoe UI&quot;;
}
#phonenumber_input:focus{
	border: 2px solid rgb(0, 170, 255);        
    background: #ffffff;  
}</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_9">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_5">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Quyền</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="role_comboBox">
                 <property name="minimumSize">
                  <size>
                   <width>130</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>-1</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QComboBox#role_comboBox {
    border: 2px solid rgb(226, 226, 226);    
    border-radius: 3px;           
    padding: 6px 10px;             
    background: #fdfdfd;           
    font: 16px &quot;Segoe UI&quot;;
}

QComboBox#role_comboBox:focus {
    border: 2px solid rgb(0, 170, 255);        
    background: #ffffff;           
}

QComboBox#role_comboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border-left: 1px solid #ccc;
}

QComboBox#role_comboBox::down-arrow {
	image: url(:/UI/icons/chevron-down.svg);
    width: 12px;
    height: 12px;
}
</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>Nhân viên</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Quản lí</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_7">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Trạng thái</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="status_comboBox">
                 <property name="minimumSize">
                  <size>
                   <width>130</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>-1</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QComboBox#status_comboBox {
    border: 2px solid rgb(226, 226, 226);    
    border-radius: 3px;           
    padding: 6px 10px;             
    background: #fdfdfd;           
    font: 16px &quot;Segoe UI&quot;;
}

QComboBox#status_comboBox:focus {
    border: 2px solid rgb(0, 170, 255);        
    background: #ffffff;           
}

QComboBox#status_comboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border-left: 1px solid #ccc;
}

QComboBox#status_comboBox::down-arrow {
	image: url(:/UI/icons/chevron-down.svg);
    width: 12px;
    height: 12px;
}
</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>Đang làm việc</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Đã nghỉ việc</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_12">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Ngày vào làm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QDateTimeEdit {
    background-color: #fdfdfd;
    border: 2px solid rgb(226, 226, 226);    
    border-radius: 6px;
    padding: 4px 6px;
    font: 16px &quot;Segoe UI&quot;;
}

QDateTimeEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #ccc;
}

QDateTimeEdit::up-button, QDateTimeEdit::down-button {
    width: 14px;
    height: 14px;
    border: none;
    background-color: transparent;
}

QDateTimeEdit::up-arrow {
    image: url(:/UI/icons/chevron-up.svg);
    width: 10px;
    height: 10px;
}

QDateTimeEdit::down-arrow {
   image: url(:/UI/icons/chevron-down.svg);
    width: 10px;
    height: 10px;
}

QDateTimeEdit:focus {
	border: 2px solid rgb(0, 170, 255);        
    background: #ffffff;  
}
QCalendarWidget {
    background-color: #fdfdfd;
    border: 1px solid #dcdcdc;
    border-radius: 8px;
}

QCalendarWidget QToolButton {
    height: 30px;
    width: 100px;
    color: #2c3e50;
    font-weight: bold;
    icon-size: 24px, 24px;
    background-color: transparent;
    border: none;
}

QCalendarWidget QToolButton:hover {
    color: #3498db;
}

QCalendarWidget QMenu {
    background-color: white;
    border: 1px solid #dcdcdc;
}

QCalendarWidget QSpinBox {
    width: 60px;
    font-size: 14px;
    padding: 2px;
}

QCalendarWidget QSpinBox::up-button, QCalendarWidget QSpinBox::down-button {
    width: 12px;
    background-color: #ecf0f1;
}

QCalendarWidget QAbstractItemView {
    outline: 0;
    selection-background-color: #3498db;
    selection-color: white;
    background-color: #ffffff;
    gridline-color: #ecf0f1;
    font-size: 13px;
}

QCalendarWidget QAbstractItemView:enabled {
    color: #2c3e50;
}

QCalendarWidget QWidget#qt_calendar_navigationbar {
    background-color: #f0f0f0;
    border-bottom: 1px solid #dcdcdc;
}

QCalendarWidget QTableView {
    border: none;
}

QCalendarWidget QTableView::item:selected {
    background-color: #3498db;
    color: white;
    border-radius: 5px;
}

QCalendarWidget QTableView::item:enabled {
    background-color: #ffffff;
}

QCalendarWidget QTableView::item:disabled {
    color: #bdc3c7;
}
</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QFrame" name="frame_15">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>32</number>
            </property>
            <item>
             <widget class="QToolButton" name="adduser_btn">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#adduser_btn{
	background-color: rgb(0, 138, 202);
	color: rgb(255, 255, 255);
	font: 15px &quot;Segoe UI&quot;;
	border-radius:5px;
}
#adduser_btn::hover{	
	background-color: rgb(0, 85, 127);
}</string>
              </property>
              <property name="text">
               <string>Thêm người dùng</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="cancel_btn">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#cancel_btn{
	background-color: rgb(255, 55, 55);
	color: rgb(255, 255, 255);
	font: 15px &quot;Segoe UI&quot;;
	border-radius:5px;
}

#cancel_btn::hover{
	
	background-color: rgb(203, 0, 0);
}</string>
              </property>
              <property name="text">
               <string>Hủy bỏ</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
